highfive/configs/rust-lang/rust.json

```diff
@@ -188,6 +193,10 @@
         "src/tools/miri": {
             "message": "Some changes occured to the Miri submodule",
             "reviewers": ["@rust-lang/miri"]
+        },
+        "library": {
+            "message": "Thank you for submitting a new PR for the library teams! If this PR contains a stabilization of a library feature that has not already completed FCP in its tracking issue, introduces new or changes existing unstable library APIs, or changes our public documentation in ways that create new stability guarantees then please comment with `r? rust-lang/libs-api @rustbot label +T-libs-api` to request review from a libs-api team reviewer. If you're unsure where your change falls no worries, just leave it as is and the reviewer will take a look and make a decision to forward on if necessary",
```

Comment:
I'm not sure I'm seeing code -- maybe we should add a test -- that the `r?` here isn't going to be picked up as a command. Triagebot has handling to ignore code blocks like this, but I don't think highfive does, so that won't be enough.

Reply:
> I'm not sure I'm seeing code -- maybe we should add a test -- that the `r?` here isn't going to be picked up as a command. Triagebot has handling to ignore code blocks like this, but I don't think highfive does, so that won't be enough.
> 
> An easy-ish fix might be to ignore things the bot itself posts, not sure if we make use of that anywhere.

Sounds good, I'll take a look and see if I can't add that.

Reply:
@Mark-Simulacrum looks like we're already good to go

https://github.com/rust-lang/highfive/blob/master/highfive/newpr.py#L438-L440